plugins/food_bot_plugin.py

```diff
@@ -74,6 +74,16 @@ def get_week_number():
         return week
 
     @staticmethod
+    def get_meal_time(meal):
+        from config import Config
```

Comment:
@andela-ydaniju please move this import statement to the top since it is being referenced in another function also.